bridges/modules/beefy/Cargo.toml

```
@@ -0,0 +1,62 @@
+[package]
+name = "pallet-bridge-beefy"
+version = "0.1.0"
+description = "Module implementing BEEFY on-chain light client used for bridging consensus of substrate-based chains."
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
```

I think we should also align all our module with this stuff with the rest of the polkadot-sdk:

```suggestion
authors.workspace = true
edition.workspace = true
```

also somewhere I can see - not sure if we should add it or not @ggwpez?
```
repository.workspace = true
```

---

Good point. Addressed here: https://github.com/paritytech/parity-bridges-common/pull/2910